app/src/main/java/com/onboarding/calculatorkotlin/MainActivity.kt

```diff
@@ -1,10 +1,12 @@
 package com.onboarding.calculatorkotlin
 
+import android.content.pm.ActivityInfo
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
+        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
```

Comment:
this is not the correct form to portrait the activity. Its more easily if you put it in the manifest

Reply:
Solved!